### DISCUSSION TITLE
Problem with second entity

### DISCUSSION THREAD

**Discussion by rtmad21** [ExperienceLovelace/ha-floorplan#114] [Q&A]
So  have been scratching my head trying to figure out what I am missing. Here is what I have:
![snip](https://user-images.githubusercontent.com/86087851/122657330-e02a6c00-d11f-11eb-8cb4-aafc730c399d.png)

When you click the lamp, the image changes correctly,  but the image for the light also changes, and its to the color of the lamp.
When you click the light, the image does nothing, but the lamp turns black. 

If I remove the coding for either, the remaining entity works perfect. I have tried everything that I can think of.

Anyone have any idea what it is that I am missing?

**Comment by OzGav** [comment]
Have you had a good look at the labels you have changed in the SVG?

**Comment by pkozul** [comment]
Hi @rtmad21 

Can you include your SVG file and CSS as well? We can then try to reproduce the same scenario to see what the cause is.
